prog/kube-peers/annotations.go

```diff
@@ -137,9 +137,7 @@ func (cml *configMapAnnotations) UpdateAnnotation(key, value string) error {
 	cm := cml.cm.DeepCopy()
 	cm.Annotations[key] = value
 	cm, err := cml.Client.ConfigMaps(cml.Namespace).Update(cm)
-	if err == nil {
-		cml.cm = cm
-	}
+	cml.cm = cm
```

Comment:
This seems to obviate the need for the DeepCopy. What am I missing?

Reply:
I think you're right.
Feels slightly strange to update something in-place when it might not go through, but I suppose comments can explain it.

Reply:
Updated.